plaidbench/plaidbench.py

```diff
@@ -55,6 +56,9 @@ def make_parser():
     parser = argparse.ArgumentParser()
     plaidargs = parser.add_mutually_exclusive_group()
     plaidargs.add_argument('--plaid', action='store_true', help="Use PlaidML as the backend.")
+    plaidargs.add_argument('--plaid-edsl',
```

Comment:
Since plaidml2 is a temporary transition, it'd be good if we could specify the `KERAS_BACKEND` from the outside and thus not require a separate `--plaid-edsl` flag.

Reply:
Should be doable